_authors/osci_2023/germainelee02.markdown

```diff
@@ -0,0 +1,9 @@
+---
+name: Germaine Lee
+short_name: germainelee02
+photo: "/assets/media/authors/germainelee02.jpg"
+twitter: nil
```

Comment:
Can you remove these two lines?

Reply:
Yes sure, I have done it. Let me know if there is anything else that needs to be done!